### PR TITLE
56 merge motors

### PR DESCRIPTION
**Pull request by qzhang234**
Transferred all motors that I can identify from 15-spec-config.py to 10-devices.py. Left comments in 10-devices.py about motors I don't recognize. Made up a few class labels in 10-devices.py.

Also I can't find pind3z and pind4z in 15-spec-config.py. @sureshnaps Did we recently stopped using those two motors?

### CODE REVIEW

File: profile_bluesky/startup/10-devices.py [hunk 1]
```diff
@@ -89,4 +96,109 @@ class SlitIpinkDevice(Device):
...
+        
+class Shutter(Device):  
+    """
+    Shutter Stage at 8-ID-I
+    """    
+    x = Component(EpicsMotor, '8idi:m1', labels=["motor", "shutter"])
+    z = Component(EpicsMotor, '8idi:m2', labels=["motor", "shutter"])
+ 
+        
+class Shutter(Device):
```
Comment (prjemian) [inline profile_bluesky/startup/10-devices.py]: Same class as above.  Copy/paste typo here?

File: profile_bluesky/startup/10-devices.py [hunk 2]
```diff
@@ -89,4 +96,109 @@ class SlitIpinkDevice(Device):
...
+
+class DetStageD(Device):  
+    """
+    Downstream detector stage at 8 m (the old 'fccdx' and 'fccdz')
+    """    
+    x = Component(EpicsMotor, '8idi:m25', labels=["motor", "det"])
+    z = Component(EpicsMotor, '8idi:m83', labels=["motor", "det"])
+
+    
+class Piezo(Device):
```
Comment (prjemian) [inline profile_bluesky/startup/10-devices.py]: Rather generic name for stage with everything specified.  Perhaps something like `SamplePiezoStage`?

File: profile_bluesky/startup/15-spec-config.py [hunk 3]
```diff
@@ -4,52 +4,23 @@
...
-ti4xd = EpicsMotor('8idi:m29', name='ti4xd', labels=("motor",))
 tthAPD = EpicsMotor('8idi:sm3', name='tthAPD', labels=("motor",))
-ccdx = EpicsMotor('8idi:m90', name='ccdx', labels=("motor",))
-fccdx = EpicsMotor('8idi:m25', name='fccdx', labels=("motor",))
-fccdz = EpicsMotor('8idi:m83', name='fccdz', labels=("motor",))
 foceye = EpicsMotor('8idi:m37', name='foceye', labels=("motor",))
-piezox = EpicsMotor('8idi:m69', name='piezox', labels=("motor",))
-piezoz = EpicsMotor('8idi:m70', name='piezoz', labels=("motor",))
+
+# What is si6?
```
Comment (prjemian) [inline profile_bluesky/startup/15-spec-config.py]: si6: Slit 6 in 8-ID-I station.  During discussions, we identified this slit is no longer in use.

File: profile_bluesky/startup/15-spec-config.py [hunk 4]
```diff
@@ -4,52 +4,23 @@
 
 flyz = EpicsMotor('8idiAERO:aero:c0:m1', name='flyz', labels=("motor",))
 # line 1: MOT001 =    NONE:2/64   2000  1  2000  200   50  125    0 0x003  StrainN  StrainN
-ti3y = EpicsMotor('8idi:m16', name='ti3y', labels=("motor",))
 tth = EpicsMotor('8idi:sm1', name='tth', labels=("motor",))
```
Comment (prjemian) [inline profile_bluesky/startup/15-spec-config.py]: What does this stage rotate about axis 2theta?  Sample, optic, detector?

File: profile_bluesky/startup/10-devices.py [hunk 5]
```diff
@@ -89,4 +96,109 @@ class SlitIpinkDevice(Device):
...
+class TI3(Device):  
+    """
+    Optics Table 3 in 8-ID-I
+    """    
+    x = Component(EpicsMotor, '8idi:TI3:x', labels=["motor", "table"])
+    y = Component(EpicsMotor, '8idi:m16', labels=["motor", "table"])  # ? Why is ti3 the only one that has a y stage?
+    z = Component(EpicsMotor, '8idi:TI3:y', labels=["motor", "table"])
+    
+
+class TI4(Device):
```
Comment (prjemian) [inline profile_bluesky/startup/10-devices.py]: use more descriptive class name:  `TableI4` or `TI4Table`

File: profile_bluesky/startup/10-devices.py [hunk 6]
```diff
@@ -89,4 +96,109 @@ class SlitIpinkDevice(Device):
...
+
+class DiamondMirror(Device):  
+    """
+    I/E Beam-splitting Mirror in 8-ID-D
+    """    
+    x = Component(EpicsMotor, '8idd:m1', labels=["motor", "optics"])
+    z = Component(EpicsMotor, '8idd:m2', labels=["motor", "optics"])
+
+
+class TI1(Device):
```
Comment (prjemian) [inline profile_bluesky/startup/10-devices.py]: see comment for TI4

File: profile_bluesky/startup/10-devices.py [hunk 7]
```diff
@@ -89,4 +96,109 @@ class SlitIpinkDevice(Device):
...
+
+class TI1(Device):  
+    """
+    Optics Table 1 in 8-ID-I
+    """    
+    x = Component(EpicsMotor, '8idi:TI1:x', labels=["motor", "table"])
+    z = Component(EpicsMotor, '8idi:TI1:y', labels=["motor", "table"])
+    
+    
+class TI2(Device):
```
Comment (prjemian) [inline profile_bluesky/startup/10-devices.py]: see comment for TI4

File: profile_bluesky/startup/10-devices.py [hunk 8]
```diff
@@ -89,4 +96,109 @@ class SlitIpinkDevice(Device):
...
+    
+class TI2(Device):  
+    """
+    Optics Table 2 in 8-ID-I
+    """    
+    x = Component(EpicsMotor, '8idi:TI2:x', labels=["motor", "table"])
+    z = Component(EpicsMotor, '8idi:TI2:y', labels=["motor", "table"])
+    
+    
+class TI3(Device):
```
Comment (prjemian) [inline profile_bluesky/startup/10-devices.py]: see comment for TI4

File: profile_bluesky/startup/10-devices.py [hunk 9]
```diff
@@ -89,4 +96,109 @@ class SlitIpinkDevice(Device):
...
-   
+class TA1(Device):  
+    """
+    Optics Table 1 in 8-ID-A
+    """    
+    x = Component(EpicsMotor, '8ida:TA1:x', labels=["motor", "table"])  # Can I make up label called 'table'?
+    z = Component(EpicsMotor, '8ida:TA1:y', labels=["motor", "table"])
+  
+
+class TA2(Device):
```
Comment (prjemian) [inline profile_bluesky/startup/10-devices.py]: see comment for TI4

File: profile_bluesky/startup/10-devices.py [hunk 10]
```diff
@@ -89,4 +96,109 @@ class SlitIpinkDevice(Device):
     hcen = Component(EpicsMotor, '8idi:SlitpinkHcenter', labels=["motor", "slit"])
     
     
-   
+class TA1(Device):
```
Comment (prjemian) [inline profile_bluesky/startup/10-devices.py]: see comment for TI4

**Comment by sureshnaps** [comment]
@prjemian @qzhang234 i am changing this file locally, will update it, grouping sample and slits and tables as per functionality.

**Comment by prjemian** [comment]
This is great.  Let me know when it is time to review.

**Comment by sureshnaps** [comment]
@prjemian i am learning git now by making mistakes. I did a commit to I think the 56-merge branch, tried to send it to you for review, did not quite go well that way. Can you please take a look?

**Comment by prjemian** [comment]
I'll take a look.  Seems that there are new commits to the master branch and those have created some conflicts to be resolved for the branch on this PR.  I made some changes but that might not have kept up with the new code in the master branch.  If you have additional changes, commit them and then I can look at the whole situation.

**Comment by prjemian** [comment]
[This](https://github.com/aps-8id-trr/ipython-8idiuser/pull/65#issuecomment-529570380) is OK.

**Comment by sureshnaps** [comment]
by mistake, i made some changes to master, but then took that changes over to 56-merge-motors branch and committed. Need your help to resolve the master conflict.

**Pull request merged**